src/components/ResponsiveTable/ResponsiveTable.tsx

```diff
@@ -0,0 +1,56 @@
+import { Theme } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import { makeStyles } from "@material-ui/styles";
+import classNames from "classnames";
+import React from "react";
+
+const useStyles = makeStyles(
+  (theme: Theme) => ({
+    [theme.breakpoints.up("sm")]: {
+      "& table": {
+        tableLayout: "fixed"
+      }
+    },
+    root: {
+      "& table": {
+        tableLayout: "auto"
+      },
+      overflowX: "auto",
+      width: "100vw"
```

Comment:
Why not just set it to `100%`? Now it just stretches outside container:
<img width="381" alt="Screenshot 2019-11-06 at 13 29 17" src="https://user-images.githubusercontent.com/6833443/68298376-7d46a700-0099-11ea-8687-536621190541.png">